ref_builder/snapshotter/otu.py

```diff
@@ -167,70 +144,61 @@ def __init__(self, path: Path):
         self._toc = OTUSnapshotToC(self.path / "toc.json")
         """The path to this snapshot's table of contents."""
 
-        self._metadata_path = self.path / "metadata.json"
-        """The path of this snapshot's metadata file."""
+        self.at_event: int | None = None
 
-        self._metadata = (
-            self._load_metadata() if self._metadata_path.exists() else OTUSnapshotMeta()
+    @classmethod
+    def new(cls, at_event: int, otu: RepoOTU, path: Path) -> "OTUSnapshot":
```

Comment:
Can this be cut? Haven't found a use for it that isn't covered by original init.

Reply:
Yup. Good call.